feat: Replaced descriptions with Hint.css tooltips

nishitlangaliya opened the pull request:
## Description
This PR will fix https://github.com/WordImpress/Google-Maps-Builder/issues/85

## How Has This Been Tested?
Manually Tested

## Video:

https://www.useloom.com/share/3df2112a0379410eb459a5de3e991cd7

## Types of changes
 New feature (non-breaking change which adds functionality)

## Checklist:
- [X] My code is tested.
- [X] My code follows the WordPress code style.
- [X] My code follows has proper inline documentation.

DevinWalker commented:
@nishitlangaliya I don't see in the PR where you are removing qTip2 from being enqueued and the actual files from the repo.

nishitlangaliya commented:
@DevinWalker ,

As per your suggestions I have reverted the core file changes for this PR and do the changes using a callback function for apply tooltip.

Regarding `removing qTip2 from being enqueued` I had already made PR for remove qTip.js and implemented hint.css ref  https://github.com/WordImpress/Maps-Builder-Core/pull/52. 

Thanks

DevinWalker commented on includes/admin/class-gmc-admin.php:
```diff
@@ -864,6 +869,140 @@ function places_search( $output ) {
+	 */
+
+	function render_maker_field_tooltip( $id ) {
+		if ( 'render_create_marker_tooltip' === $id ) {
```
@nishitlangaliya change this to a switch/case for easier readability.

nishitlangaliya commented:
@DevinWalker ,

Thanks for the suggestion. As per suggestions I have made the changes and converted if else statement to switch case. Please review it and let me know if any changes required.

Thanks
